packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.types.ts

```diff
@@ -274,4 +277,10 @@ export interface IGroupDividerProps {
 
   /** Determines if the group selection check box is shown for collapsed groups. */
   isCollapsedGroupSelectVisible?: boolean;
+
+  /** Allows the user to provide custom class name for the header chevron. */
+  headerChevronClassName?: string;
+
+  /** Override which allows the caller to provider a custom title. */
+  onRenderTitle?: (group: IGroup) => React.ReactNode;
```

Comment:
can we use IRenderFunction here and provide the originalRender method to the onRenderTitle prop.